data.sql

```diff
@@ -48,3 +48,181 @@ WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
+SELECT id FROM animals WHERE name = 'Plantmon';
+
+-- Plantmon visited William Tatcher on Aug 10th, 2020.
+SELECT id FROM vets WHERE name = 'William Tatcher';
+
+-- Plantmon visited Maisy Smith on Apr 7th, 2021.
+-- Additional visit for Plantmon and Maisy Smith will be handled later.
+
+-- Squirtle visited Stephanie Mendez on Sep 29th, 2019.
+SELECT id FROM animals WHERE name = 'Squirtle';
+
+-- Angemon visited Jack Harkness on Oct 3rd, 2020.
+SELECT id FROM animals WHERE name = 'Angemon';
+
+-- Boarmon visited Maisy Smith on Jan 24th, 2019.
+SELECT id FROM animals WHERE name = 'Boarmon';
+
+-- Blossom visited Stephanie Mendez on May 24th, 2020.
+SELECT id FROM animals WHERE name = 'Blossom';
+
+-- Blossom visited William Tatcher on Jan 11th, 2021.
+SELECT id FROM vets WHERE name = 'William Tatcher';
+
+
+
+-- Agumon: ID = 1
+-- Gabumon: ID = 2
+-- Pikachu: ID = 3
+-- Devimon: ID = 4
+-- Charmander: ID = 5
+-- Plantmon: ID = 6
+-- Squirtle: ID = 7
+-- Angemon: ID = 8
+-- Boarmon: ID = 9
+-- Blossom: ID = 10
+-- Vet William Tatcher: ID = 1
+-- Vet Stephanie Mendez: ID = 3
+-- Vet Jack Harkness: ID = 4
+-- Vet Maisy Smith: ID = 2
+
+-- Agumon visited William Tatcher on May 24th, 2020.
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES (1, 1, DATE '2020-05-24');
+
+-- Agumon visited Stephanie Mendez on Jul 22nd, 2020.
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES (1, 3, DATE '2020-07-22');
+
+-- Gabumon visited Jack Harkness on Feb 2nd, 2021.
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES (2, 4, DATE '2021-02-02');
+
+-- Pikachu visited Maisy Smith on Jan 5th, 2020.
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES (3, 2, DATE '2020-01-05');
+
+-- Pikachu visited Maisy Smith on Mar 8th, 2020.
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES (3, 2, DATE '2020-03-08');
+
+-- Pikachu visited Maisy Smith on May 14th, 2020.
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES (3, 2, DATE '2020-05-14');
+
+-- Devimon visited Stephanie Mendez on May 4th, 2021.
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES (4, 3, DATE '2021-05-04');
+
+-- Charmander visited Jack Harkness on Feb 24th, 2021.
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES (5, 4, DATE '2021-02-24');
+
+-- Plantmon visited Maisy Smith on Dec 21st, 2019.
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES (6, 2, DATE '2019-12-21');
+
+-- Plantmon visited William Tatcher on Aug 10th, 2020.
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES (6, 1, DATE '2020-08-10');
+
+-- Plantmon visited Maisy Smith on Apr 7th, 2021.
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES (6, 2, DATE '2021-04-07');
+
+-- Squirtle visited Stephanie Mendez on Sep 29th, 2019.
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES (7, 3, DATE '2019-09-29');
+
+-- Angemon visited Jack Harkness on Oct 3rd, 2020.
+INSERT INTO visits (animal_id, vet_id, visit_date)
+VALUES (8, 4, DATE '2020-10-03');
+
```

Comment:
- [ ] @karanJ2212 great work adding data to your database however you forgot to add Jack visit on November 4th, kindly consider adding it
![image](https://github.com/karanJ2212/vet_clinic/assets/61408860/39fbf441-e291-42e8-9928-8cc380b8c932)

Reply:
done

Reply:
Please note that this change has not been implemented yet. Kindly look into the dates and make the necessary addition 👍🏽

Reply:
<img width="689" alt="image" src="https://github.com/karanJ2212/vet_clinic/assets/107232162/5908e94c-6ef4-4553-b8da-d3be60c0d0a7">
hii @rotimiazeez  I can see the highlighted data is there in the table. Can you tell me how are you checking the data?

Reply:
My apologies @karanJ2212, this is a mix-up from my end. 🙏🏽